### DISCUSSION TITLE
收集：“号称”自主研发的开源套壳

### DISCUSSION THREAD

**Discussion by LinuxSuRen** [LinuxSuRen/open-source-best-practice#148] [General]
案例：CEC-IDE

* 非常明显的 VS-Code 风格
* [microsoft/vscode/issues/191229](https://github.com/microsoft/vscode/issues/191229)
* https://cecide.digitalgd.com.cn/
* [23年8月26日，官方致歉申明](https://mp.weixin.qq.com/s/z-OOz7tai_loDWBNXkEm_Q)

![image](https://github.com/LinuxSuRen/open-source-best-practice/assets/1450685/df20e825-5b5a-4f81-b9d1-34115dc2442a)
![cd4a1f7ad5c1ebae6441ac9fcff75d7](https://github.com/LinuxSuRen/open-source-best-practice/assets/1450685/d88228c5-9de1-4e6d-a628-fd7c0ad7c5db)
![image](https://github.com/LinuxSuRen/open-source-best-practice/assets/1450685/fd4b0fb4-8a7f-4a59-b183-ce82b347c07b)
![image](https://github.com/LinuxSuRen/open-source-best-practice/assets/1450685/48f02588-c2b6-4e14-a68b-147212e08a0e)
![e74e2bfed483e2f047ceaef95fae8d8](https://github.com/LinuxSuRen/open-source-best-practice/assets/1450685/92632c12-a730-4b46-ad0e-51ad844041a6)

8/25日，中午12点左右，网站无法访问

![image](https://github.com/LinuxSuRen/open-source-best-practice/assets/1450685/4ee4d3d2-8abf-473b-801b-f679e55a89d7)

**Comment by addozhang** [comment]
重新定义“自主研发”

**Reply by LinuxSuRen** [reply]
这种愚蠢的做法，就应该永远记录下来，希望后来者能避免。

**Comment by zhengkai** [comment]
倒数第二个截图里的 微软copyright 是在这个 cec-ide 的 console 里看到的么，太吊了……

**Reply by LinuxSuRen** [reply]
是的，昨天晚上下载后看到的。大家可以试试，说不定他们什么时候发现就修改了。
